assets/pages/bundles-landing/helpers/abtest.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Participation } from 'helpers/abtest';
+
+
+// ----- Functions ----- //
+
+const otherWaysOfContribute = (participation: Participation): string => {
+
+  const variant = participation.otherWaysOfContribute;
+
+  switch (variant) {
+    case 'control' : return 'other ways you can support us';
+    case 'variantA' : return 'other ways you can contribute';
+    case 'variantB' : return 'other ways you can give us money';
```

Comment:
Another approach for reducing the amount of code could be creating a shared presentation module and leave the ab-test.js module as simple as possible. This kind of shared component would be justified in the case of a test which impacts several pages. In the example test, we created a non-shareable presentation module for the a/b test which only impacts the bundle landing page..